sound/soc/intel/boards/sof_pcm512x.c

```diff
@@ -316,11 +323,15 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 		if (!links[id].cpus->dai_name)
 			goto devm_err;
 
-		idisp_components[i - 1].name = "ehdaudio0D2";
-		idisp_components[i - 1].dai_name = devm_kasprintf(dev,
-								  GFP_KERNEL,
-								  "intel-hdmi-hifi%d",
-								  i);
+		if (idisp_codec) {
```

Comment:
@kv2019i looks good now. Can we please add the same comment from the commit message about why we need to set the dummy comps here as well?

Reply:
@ranj063 Ack, added to next revision.